### PR TITLE
Updated Delete Dialog

### PR DESCRIPTION
**Pull request by Hacker437**
Issue : #792
PR continution : #815 
Updated Strings to get the correct dialog

I am sorry for making a mess in this repo by making double PR for 2 times

### CODE REVIEW

File: app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java [hunk 1]
```diff
@@ -71,11 +71,15 @@ public static void showDeleteEntriesDialog(Activity activity, List<VaultEntry> s
         TextView textMessage = view.findViewById(R.id.text_message);
         TextView textExplanation = view.findViewById(R.id.text_explanation);
         String entries = services.stream()
-                .map(entry -> !entry.getIssuer().isEmpty() ? entry.getIssuer()
-                        : !entry.getName().isEmpty() ? entry.getName()
-                        : activity.getString(R.string.unknown_issuer)
+                .map(entry -> !entry.getIssuer().isEmpty() && !entry.getName().isEmpty()
```
Comment (alexbakker) [inline app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java:74]: This has become kind of unreadable. Please move it to a function.

**Review by alexbakker — CHANGES_REQUESTED** [review]
This looks a little messy now:

image

The "To prevent losing access" paragraph should be on a new line and not have any spaces in front of it. Please remove the comma's and dots behind the entries in the list.

Changes you make to any translated strings.xml files will be overwritten by Crowdin, so please remove them.

File: app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java (hunk 1 above)
Comment (Hacker437) [reply app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java:74]: Sure

**Comment by Hacker437** [comment]
> Changes you make to any translated strings.xml files will be overwritten by Crowdin, so please remove them.

Ok i will change them

**Comment by alexbakker** [comment]
I pushed some changes to your branch to fix some minor layout and code style issues. You're still set as the commit author, so your work will still be credited to you. Thanks for the help!

**Review by alexbakker — APPROVED** [review]

**Pull request merged**